simplified-ui-settings/src/main/java/org/nypl/simplified/ui/settings/SettingsFragmentMain.kt

```diff
@@ -87,6 +88,14 @@ class SettingsFragmentMain : PreferenceFragmentCompat() {
   }
 
   private fun configureVersion(preference: Preference) {
+    try {
+      val context = requireContext()
+      val pkgManager = context.packageManager
+      val pkgInfo = pkgManager.getPackageInfo(context.packageName, 0)
+      preference.summary = "${pkgInfo.versionName} (${pkgInfo.versionCode})"
+    } catch (e: PackageManager.NameNotFoundException) {
+      //  Pass
```

Comment:
If an exception is thrown the version won't be shown (no different than the existing behavior).